floris/simulation/base.py

```diff
@@ -54,7 +63,7 @@ def get_model_defaults(cls) -> Dict[str, Any]:
         Dict[str, Any]
             Dictionary of keyword argument: default.
         """
-        return {el.name: el.default for el in attrs.fields(cls)}
+        return {el.name: el.default for el in fields(cls)}
 
     def _get_model_dict(self) -> dict:
```

Comment:
Good question - this is only currently used in `floris.tools.interface_utilities.show_params`, but that function doesn't work for other reasons. Since that function can use the model's `as_dict` method, like you mentioned, I'll remove this `_get_model_dict` function from here.